tests/decorator.test.ts

```diff
@@ -1,157 +0,0 @@
-import { PoolClient } from "pg";
```

Comment:
Why are we deleting this?  Redundant?

Reply:
Since we move everything to use decorators, this file is no longer useful.